opflexagent/vif_plug_vpp/vpp.py

```diff
@@ -0,0 +1,154 @@
+# Derived from nova/virt/libvirt/vif.py
+#
+# Copyright (C) 2011 Midokura KK
+# Copyright (C) 2011 Nicira, Inc
+# Copyright 2011 OpenStack Foundation
```

Comment:
Make the licensing consistent with other files

Reply:
Sure